src/main/java/at/hannibal2/skyhanni/features/inventory/ItemStars.kt

```diff
@@ -23,7 +25,7 @@ class ItemStars {
 
         val stack = event.itemStack ?: return
         if (stack.stackSize != 1) return
```

Comment:
Why are they all separate ifs?

Reply:
Better readability. But this was an older code style. I would try to merge them together now or extract into additional functions

Reply:
ah i see